Fully update first 3D Shader page (#10554)
godotengine/godot-docs: tutorials/shaders/your_first_shader/your_first_3d_shader.rst

Add a new :ref:`MeshInstance3D <class_MeshInstance3D>` node to your scene.

> In the inspector tab beside "Mesh" click "<empty>" and select "New PlaneMesh".

Reviewer: ```suggestion
In the inspector tab, set the **Mesh** property to a new PlaneMesh.
```
I'm not a fan of how this page spells out how to create new resources by `clicking on "<empty>"` each time. I feel like a shader tutorial should expect a little more familiarity with the inspector UI than that?

Author: This is the only page I can think of that actually describes clicking on "<empty>"

Reviewer: Search the page for "empty" and it comes up at least four times

Reviewer: ![Code_WCtdPb9psl](https://github.com/user-attachments/assets/1b78814b-1d17-432d-9320-c57ba1f93810)
It comes up a couple other times. I think it's fine and justified in the Getting Started tutorials, since it makes it extremely clear what to do. My guess is that it's a stylistic convention that was being followed at the time these pages were written.

I think we should move towards describing cases like this as "setting the Whatever property to a new Thing". If necessary we could also add in a general tutorial somewhere in the docs (either in Getting Started or maybe in the Inspector page) about which buttons to click to set properties, so this is explained in one place rather than piecemeal wherever it comes up. Something for another PR.

.. image:: img/normal-set.webp

When we have normals that correspond to a specific vertex we set ``NORMAL``, but
if you have a normalmap that comes from a texture, set the normal using
``NORMAL_MAP`` in the ``fragment()`` function. This way Godot will handle the

> wrapping of texture around the mesh automatically.

Reviewer: The section on setting the normals *is* a little broken, as indicated in https://github.com/godotengine/godot-docs/pull/10501. I think that PR suggests the wrong fix, though.

I believe the intention with the tutorial is that the plane mesh normals are to be corrected by reading from a normal map. But since the normal map texture has high resolution, and the height is changed in `vertex()` but the normals are changed in `fragment()`, it not only corrects for the height, but also adds a bunch of extra normal detail. I *think* the right solution here might be to set the NORMAL value in `vertex()` rather than setting a normal map here. I'm not 100% sure, though.

> ``NORMAL_MAP`` in the ``fragment()`` function. This way Godot will handle the
> wrapping of texture around the mesh automatically.

Reviewer 2: ```suggestion
``NORMAL_MAP`` in the ``fragment()`` function. This way Godot will handle
wrapping the texture around the mesh automatically.
```
Or

```suggestion
``NORMAL_MAP`` in the ``fragment()`` function. This way Godot will handle the
wrapping of the texture around the mesh automatically.
```
